todo_app.sql

```diff
@@ -0,0 +1,77 @@
+DROP TABLE IF EXISTS tasks;
+
+CREATE TABLE tasks
+(
+  id serial NOT NULL primary key,
+  title varchar(255) NOT NULL,
+  description text,
+  created_at timestamp
+  without time zone NOT NULL DEFAULT now
+  (),
+  updated_at timestamp without time zone,
+  completed boolean NOT NULL DEFAULT false
+);
+
+  ALTER TABLE tasks DROP completed;
+  ALTER TABLE tasks DROP updated_at;
```

Comment:
So I wouldn't drop the column here. It's best to find a way to just add the NOT NULL and DEFAULT now() constraints.